src/__tests__/integration/settings.test.js

```diff
@@ -0,0 +1,54 @@
+import { integrationTest, waitFor, uploadFile, getNextStepButton, getTestFile } from './index';
+import { act } from 'react-dom/test-utils';
+import React from 'react';
+import Settings from '../../components/layouts/pages/settings/Settings';
+
+const TEST_FILE = getTestFile();
+const PROFILE_VALUE = 'TAGGED_PDF';
+
+const isFileUploaded = state => state.pdfFiles.length;
+
+const getProfileValue = store => store.getState().jobSettings.profile;
+const simulateProfileChange = (component, value) =>
+    component
+        .find('#jobProfile')
+        .at(0)
+        .props()
+        .onChange({ target: { value } }, null);
+
+describe('Settings', () => {
+    it(
+        'Settigns page loaded',
+        integrationTest(async (store, component) => {
+            // Wrapped in act() cause of warning: 'When testing, code that causes React state updates should be wrapped into act(...'
```

Comment:
Let's wrap repeated preparation part:
```
await act(async () => {
    await uploadFile(component, TEST_FILE);
    await waitFor(store, isFileUploaded);
});
component.update();
```
into single utility `uploadFile`. Current `uploadFile` utility can be renamed to `selectFile`.

Reply:
Fixed